src/js/me-i18n-locale-en.js

```diff
@@ -33,7 +33,7 @@
             'mejs.fullscreen' : 'Fullscreen',
 
             // mep-feature-jumpforward
-            'mejs.time-jump-forward': 'Jump forward %1 seconds',
+            'mejs.time-jump-forward': 'Jump forward {0} {0|plural:second:seconds}',
```

Comment:
In your experience, what could be a more general format? The only thing I want to avoid is to overcomplicate this given that the translations we use with plurals are really minimal, but I also want to follow standards as much as possible